vm/compiler/src/ledger/get.rs

```diff
@@ -105,6 +105,11 @@ impl<N: Network, B: BlockStorage<N>, P: ProgramStorage<N>> Ledger<N, B, P> {
             None => bail!("Missing signature for block {height}"),
         }
     }
+
+    /// Returns the record for the given `commitment`.
```

Comment:
I will just comment for future consideration that these pass-through methods are generally a bad sign, suggesting that there may be room for refactoring how the structs are nested and accessed by the client code (not part of this PR or functionality, but something that we need to consider for improvement in the future).